### DISCUSSION TITLE
Log custom variable at Logging model events

### DISCUSSION THREAD

**Discussion by igorespin** [spatie/laravel-activitylog#1006] [Q&A]
Hi!

I'm logging model events, the question is: Is there a way to have a custom variable so I can log it in tapActivity function? 

Maybe set a value to a temporary variable on the model before it calls the save method, so I can use it in tapActivity function?

Thanks!

**Comment by Gummibeer** [comment]
Yes it's possible, all classes are plain PHP - so you can add properties and use them as you like.
